lib/formulaic/label.rb

```diff
@@ -20,7 +27,21 @@ def to_str
     private
 
     def translate
-      I18n.t(lookup_paths.first, scope: :'simple_form.labels', default: lookup_paths).presence
+      translations.detect(&:present?)
```

Comment:
You could save a loop with a `#reduce`!

Something like:

```ruby
Formulaic.translation_scopes.reduce(nil) do |result, scope|
  result ||
    lookup_translation([model_name, action, attribute], scope: scope) ||
    lookup_translation([model_name, attribute], scope: scope) ||
    lookup_translation(attribute, scope: scope)
end
```

Unsure whether that's still legible, of course.